example/src/utils/fetch.ts

```diff
@@ -0,0 +1,29 @@
+import { ISSUER_AUTH_TOKEN, ISSUER_BASE_URL } from "@env";
+
+function addAuthHeaders(
+  options: RequestInit,
+  authHeaders: Record<string, string>
+) {
+  return {
+    ...options,
+    headers: {
+      ...options.headers,
+      ...authHeaders,
+    },
+  };
+}
+
+export default function appFetch(request: RequestInfo, options: RequestInit) {
+  const requestUrl =
+    typeof request === "string" ? new URL(request) : new URL(request.url);
+  let authHeaders: Record<string, string> = {};
+
+  // Add the authentication header only if I am contacting the issuer URL
+  if (requestUrl.origin === new URL(ISSUER_BASE_URL).origin) {
+    authHeaders = {
+      Authorization: `${ISSUER_AUTH_TOKEN}`,
+    };
+  }
```

Comment:
```suggestion
  const authHeaders = requestUrl.origin === issuerBaseUrl.origin 
    ? { Authorization: `Bearer ${issuerAuthToken}` } 
    : {};
```
Does it make sense to you?

Reply:
Yes, much better. I'm only eliminating Bearer because they told us he won't be there 🫢